pkg/eval/json_evaluator.go

```diff
@@ -29,6 +30,17 @@ const (
 	Disabled = "DISABLED"
 )
 
+func NewJSONEvaluator(logger *logger.Logger) *JSONEvaluator {
+	ev := JSONEvaluator{
+		Logger: logger.WithFields(
+			zap.String("component", "evaluator"),
+			zap.String("evaluator", "json"),
+		),
+	}
+	jsonlogic.AddOperator("fractionalEvaluation", ev.fractionalEvaluation)
```

Comment:
Adding the operator multiple times is redundant I believe, I think that's why I put it in an `init` to begin with

Reply:
I moved it here to provide it access to the same loggers as the rest of the packages - it seemed to have a specific function signature so i felt this was the cleanest way to handle it.
Unless im missing something, happy to refactor